### PR TITLE
Bug/app rerender rating submission

### PR DESCRIPTION
**Pull request by b-stein**
#### What's this PR do?
fixes the bug so app rerender rating submission and refresh the movie card + initial user login loads the correct movie cards

#### Where should the reviewer start?
files changed tab on GH

#### How should this be manually tested?
on application DOM

#### Any background context you want to provide?
the main issue with both of these bugs was that we had 1 api call setting off another api call (from App), but the 2nd api call set off wasn't attached to an async/await in the child component, so it was rerendering without the accurate data. i left a few annotations on files changed tab

### CODE REVIEW

File: src/LoginPage/LoginPage.js [hunk 1]
```diff
@@ -22,14 +22,16 @@ class LoginPage extends Component {
...
-        });
-        this.props.getUsersRatings(data.user.id);
-        this.props.history.push(`/users/${this.state.id}`);
-      })
+				})
+				return data;
+			})
+			.then(async (data) => {
+				await this.props.getUsersRatings(data.user.id);
+				this.props.history.push(`/users/${this.state.id}`)})
```
Comment (b-stein) [inline src/LoginPage/LoginPage.js:34]: this .then() block is calling on an api call from App. so we need an async await for it to run synchronously + capture the data on the load

File: src/MovieCard/MovieCard.js [hunk 2]
```diff
@@ -4,42 +4,48 @@ import { Link, Redirect } from "react-router-dom";
...
+			this.checkForUserRating();
+		}
+		catch(e){
+			console.log(e);
+		}
+		// postRating(Number(this.state.rating), this.props.id, this.props.userID)
+		// 	.then(() => {this.props.getUsersRatings(this.props.userID)})
+		// 	.then(this.checkForUserRating())
+		// 	.catch((error) => this.setState({ error }));
+
```
Comment (b-stein) [inline src/MovieCard/MovieCard.js:35]: my friend wrote this in try/catch, but i left our .then's() so we can convert it back. we should only use try/catch if we understand it.. lets keep it if we get it though!

File: src/MovieCard/MovieCard.js [hunk 3]
```diff
@@ -4,42 +4,48 @@ import { Link, Redirect } from "react-router-dom";
...
 
-	submitRating = (event) => {
+	submitRating = async (event) => {
 		event.preventDefault();
-		postRating(Number(this.state.rating), this.id, this.userID)
-			.then(() => {this.getUsersRatings(this.userID)})
-			.catch((error) => this.setState({ error }));
+		try {
+			await postRating(Number(this.state.rating), this.props.id, this.props.userID)
+			await this.props.getUsersRatings(this.props.userID);
```
Comment (b-stein) [inline src/MovieCard/MovieCard.js:25]: similar situation -- we need to await the fetch call coming from App so our data is in sync w/ page loads

File: src/MovieCard/MovieCard.js (hunk 2 above)
Comment (attom2) [reply src/MovieCard/MovieCard.js:35]: I wrote a few the other day so I am okay with it

File: src/LoginPage/LoginPage.js (hunk 1 above)
Comment (attom2) [reply src/LoginPage/LoginPage.js:34]: Such a simple change!!

File: src/MovieCard/MovieCard.scss [hunk 4]
```diff
@@ -28,7 +28,9 @@
 
 .rating-form {
 	height: 100px;
-	width: 27vw;
+	width: 33vw;
+	min-width: 300px;
```
Comment (attom2) [inline src/MovieCard/MovieCard.scss:32]: Once again jealous of your css knowledge

**Pull request merged**